### PR TITLE
Completed javascript-conditionals

### PR DESCRIPTION
**Pull request by amorabryan**

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH >= 8 && pH <= 14) {
+    return 'base';
+  } else if (pH <= 6 && pH >= 0) {
+    return 'acid';
+  } return 'invalid pH level';
+}
```
Comment (kurtyazdizadeh) [inline javascript-conditionals/main.js]: what happens if `pH` equals `7.1` or `6.9`?

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
Let's check for the edge cases mentioned for the pH function

File: javascript-conditionals/main.js (hunk 1 above)
Comment (amorabryan) [reply javascript-conditionals/main.js]: I have altered the code to include the edge cases. Thank you for looking out!

Comment (amorabryan) [reply javascript-conditionals/main.js]: In this case, it returned with the `invalid pH level`

**Review by kurtyazdizadeh — APPROVED** [review]
Approved! 👍

**Pull request merged**